backend/src/main/resources/templates/Rhosak/scheduledUpgradeBody.html

```diff
@@ -0,0 +1,82 @@
+{#include Rhosak/rhosakEmailBody}
+{#content-title}Upgrade Notification - {action.context.upgrade_time.toUtcFormat()}{/content-title}
+{#content-body}
+<tr>
+    <td class="rh-content__block">
+        <div>Hello {user.firstName} {user.lastName},</div>
+        <p>This notification is for your {action.events.size()} OpenShift Streams {#if action.events.size() is 1} instance {#else} instances{/if} below.</p>
+        <ul>
+            {#each action.events}
+            <li style="font-size: 14px;">
+                {it.payload.name}
+            </li>
+            {/each}
+        </ul>
+
+        Your {#if action.events.size() is 1} instance is {#else} instances are{/if} scheduled for upgrade maintenance to {action.context.kafka_version} on {action.context.upgrade_time.toUtcFormat()}
```

Comment:
```suggestion
        Your {#if action.events.size() is 1} instance is {#else} instances are{/if} scheduled for upgrade maintenance to {action.context.kafka_version} Kafka version on {action.context.upgrade_time.toUtcFormat()}
```

Reply:
Might be good to call out that it is a Kafka version upgrade.

Reply:
Makes total sense, thanks.